server_installation/topics/cache/replication.adoc

```diff
@@ -1,7 +1,8 @@
 
 === Replication and Failover
 
-The `sessions`, `authenticationSessions`, `offlineSessions` and `loginFailures` caches are the only caches that may perform replication.  Entries are
+There are some caches like `sessions`, `authenticationSessions`, `offlineSessions`, `loginFailures` and few others, which are configured
+as distributed caches in case of clustered setup.  Entries are
```

Comment:
Again, if details are available, it would be nice to replace "and a few others" with those details. If no, this rewrite will be sufficient.

There are caches like `sessions`, `authenticationSessions`, `offlineSessions`, `loginFailures` and a few others, which are configured as distributed caches when using a clustered setup. Entries are...

Reply:
@matthewhelmke Well, I am not sure. Explicitly mention all the caches is more clear, but it has also one disadvantage.  Always when we add new cache into the configuration, we will also need to update this documentation with all the references to cache names. If you write "and few others", you explicitly mention to admin that there are more caches and he will quickly figure out by looking  to the configuration file what the caches are... :) WDYT? For now, I've just used the sentence you mentioned with "a few others" in it.

Reply:
See my comment above. It is not always an easy balance, and you probably know better than I what our users must know versus what they would like to have as reference.